rct229/rules/section22/section22rule26.py

```diff
@@ -0,0 +1,100 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.baseline_systems.baseline_system_util import HVAC_SYS
+from rct229.ruleset_functions.get_baseline_system_types import get_baseline_system_types
+from rct229.ruleset_functions.get_primary_secondary_loops_dict import (
+    get_primary_secondary_loops_dict,
+)
+from rct229.schema.config import ureg
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import CalcQ
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
+    HVAC_SYS.SYS_11_1,
+    HVAC_SYS.SYS_11_2,
+    HVAC_SYS.SYS_11_1B,
+]
+
+REQUIRED_PUMP_POWER_PER_FLOW_RATE = 12 * ureg("W/gpm")
+
+
+class Section22Rule26(RuleDefinitionListIndexedBase):
+    """Rule 26 of ASHRAE 90.1-2019 Appendix G Section 22 (Hot water loop)"""
+
+    def __init__(self):
+        super(Section22Rule26, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section22Rule26.PrimaryCoolingFluidLoop(),
+            index_rmr="baseline",
+            id="22-26",
+            description="For chilled-water systems served by chiller(s) and serves baseline System-11, the baseline building constant-volume primary pump power shall be modeled as 12 W/gpm.",
+            rmr_context="ruleset_model_instances/0",
+            list_path="fluid_loops[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmi_b = context.baseline
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
+        # create a list contains all HVAC systems that are modeled in the rmi_b
+        available_type_lists = [
```

Comment:
I suggest calling this variable `available_types`, `available_types_list`, or `available_sys_types`. The plural on `list` is misplaced since there is only one list here.

Reply:
update to available_sys_types